### PR TITLE
feat: DRY "Prerequisite Material" cards

### LINKED ISSUE #1546: Refactor "Helpful Resources" and "Prerequisite Material" cards (DRY)
The cards in the **Helpful Resources** and the **Prerequisite Material** sections are very similar. There is potential to DRY some code here.

The card component use in **Helpful Resources** is **AppDescriptionCard**. That component should be made reusable by **Prerequisite Material**.

### PR DESCRIPTION
**Pull request by eddybrando**
Closes #1546

### CODE REVIEW

**Comment by eddybrando** [comment]
@y4izus rebase cleanup is done 🙂

File: components/textbook-demo/PrerequisiteMaterialSection.vue [hunk 1]
```diff
@@ -4,32 +4,21 @@
       Prerequisite material
     </h2>
     <div class="prerequisite-material-section__cards">
-      <article
+      <div
         v-for="{ title, description, progress, segment, url } in prerequisites"
         :key="title"
         class="prerequisite-material-section__card"
```
Comment (y4izus) [inline components/textbook-demo/PrerequisiteMaterialSection.vue:10]: this is the container for the cards, so, to improve readability and responsibility separation, this container should have the styles:
```
    align-items: strech;
    display: flex;
    padding: $spacing-05 $spacing-05 0;
```

and the card itself should have:
```
    background: $cool-gray-10;
    color: $cool-gray-80;
```

**Review by y4izus — CHANGES_REQUESTED** [review]
Great Eddy! just one comment :)

File: components/textbook-demo/PrerequisiteMaterialSection.vue (hunk 1 above)
Comment (eddybrando) [reply components/textbook-demo/PrerequisiteMaterialSection.vue:10]: Good catch! That wrapping level is unnecessary. I can move the style and `v-for` to the child component and get the same results indeed. 🙂

**Comment by eddybrando** [comment]
@y4izus thanks a lot for the code review! I've implemented the requested changes. 🙂

**Review by y4izus — APPROVED** [review]
Thanks!

**Pull request merged**